ui/src/utils/createStackdriverUrl.js

```diff
@@ -0,0 +1,36 @@
+const stackdriverAPI = "https://console.cloud.google.com/logs/viewer";
```

Comment:
This file is almost an exact copy of this file from Merlin: https://github.com/caraml-dev/merlin/blob/main/ui/src/utils/createStackdriverUrl.js

Reply:
can add these as a comment in the code? Ref:url....

Reply:
Okay just added that!